JavaScript/Generating-sequences.md

```diff
@@ -0,0 +1,68 @@
+# Generating-sequences
+
+Mathematical sequences occur so frequently in nature very often this transfers into computer programming problems. For example, the 
+cicadas who appear periodically but only emerge after a prime number of years.  Or sunflowers which have either 34, 55 or 89 petals 
+on their face, depending upon the size of the sunflower. Additionally, lilies and irises have three petals, buttercups and wild roses 
+have five, delphiniums have eight petals, all related to the Fibonacci sequence. Or simply triangular numbers in bowling! 
+We are surrounded by sequences which are regularly analysed within computer programming.
+
+Being equipped to generate such sequences is vital. This is dedicated to exploring the possibleways these sequences can be produced 
```

Comment:
- add a noun after the word 'This' e.g. 'This article is dedicated to...'
- add space between 'possible' and 'ways'